console/src/test/java/com/arcadedb/console/ConsoleTest.java

```diff
@@ -523,4 +523,24 @@ public void testPercentWildcardInQuery() throws IOException {
       assertThat(buffer.toString().contains("Miner")).isTrue();
     }
   }
+
```

Comment:
Just pushed another test that doesn't pass (ConsoleTest.testDuplicateEntries test method). This mean the original issue is still here.